packages/server/src/queue/queue.service.ts

```diff
@@ -18,6 +18,7 @@ export class QueueService {
     });
 
     await queue.addQueueTimes();
+    await queue.checkIsOpen();
```

Comment:
This totally works but it might be slightly more clear to append the `queue.isOpen` attribute after this line like it's done with `queue.questions`, instead of doing in the `queue.checkIsOpen` function